bigdata-test/src/main/java/io/tidb/bigdata/test/ConfigUtils.java

```diff
@@ -38,6 +39,8 @@ public class ConfigUtils {
 
   public static final String tidbPassword = getEnvOrDefault(TIDB_PASSWORD, "");
 
+  public static final String telemetryEnable = getEnvOrDefault(TELEMETRY_ENABLE, "false");
```

Comment:
true?

Reply:
I think **in test** it's going to be `false` by default. right? @Daemonxiao

Reply:
This is the test default configuration. To avoid sending telemetry messages in every test, We should add false here.